MegaSolution/Transliteration/DownloadOfTrasliterationRules.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Transliteration
+{
+  class DownloadOfTransliterationRules
+  {
+    const string TRANSLITRULESRUEN = @"C:\\Users\\SATELLITE\\HW_TAT_summer\\MegaSolution\\Transliteration\\TranslitRulesRU-EN.txt";
+    const string TRANSLITRULESENRU = @"C:\\Users\\SATELLITE\\HW_TAT_summer\\MegaSolution\\Transliteration\\TranslitRulesEN-RU.txt";
+    enum codes { RuEn, EnRu};
+    /// <summary>
+    /// This method reads the rules of transliteration from .txt files and creates a dictionary
+    /// </summary>
+    /// <param name="code">
+    /// The number whitch helps to choose file to read
+    /// </param>
+    /// <returns>
+    /// Dictionary to work with
+    /// </returns>
+    public Dictionary<string, string> CreateDictionary (int code)
```

Comment:
Bad name, what dictionary? better name CreateTransliterationDictionary or something like that, more clear